instruqt-tracks/terraform-cloud-aws-v2/13-private-module-registry/assignment.md

```diff
@@ -105,13 +105,11 @@ timelimit: 1800
     font-size: 12px;
     font-weight: bold;
     align-items: center;
-    justify-content: center;
```

Comment:
The presence of these two lines in this block is different between different versions of the track.  In AWS and Azure, the `align-items` line is present; in GCP the `justify-content` line is present.